controller/controller_test.go

```diff
@@ -424,6 +424,17 @@ func (nr *NopReconciler) Reconcile(context.Context, string) error {
 	return nil
 }
 
+type testRateLimiter struct {
+	t     *testing.T
+	delay time.Duration
+}
+
+func (t testRateLimiter) When(interface{}) time.Duration { return t.delay }
+func (t testRateLimiter) Forget(interface{})             {}
+func (t testRateLimiter) NumRequeues(interface{}) int    { return 0 }
+
+var _ workqueue.RateLimiter = &testRateLimiter{}
```

Comment:
```suggestion
var _ workqueue.RateLimiter = (*testRateLimiter)(nil)
```
is usually how we roll.

Reply:
Cool! I like it.